### PR TITLE
1.7.1

### PR DESCRIPTION
**Pull request by israelpoli**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,10 +35,9 @@
 * Fixed an issue where running in a private gitlab repo caused a warning message to be shown multiple times.
 * Added a new validation to the **validate** command to verify that markdown and python files do not contain words related to copyright section.
 * Fixed an issue where **lint** crashed when provided an input file path (expecting a directory).
-* Added a new validation to **validate**, making sure all inputs of a playbook are used.
-* Added a new validation to **validate**, making sure all inputs used in a playbook declared in the input section.
```
Comment (dorschw) [inline CHANGELOG.md:32]: were these misplaced here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,7 +54,6 @@
 * Added a new flag to **generate-docs** command, allowing to add a custom image link to a playbook README.
 * Added a new validation to the **validate** command to verify that the package directory name is the same as the files contained in the that package.
 * Added support in the **unify** command to unify a schema into its Modeling Rule.
-* The **format** command will now replace the *fromServerVersion* field with *fromVersion*.
```
Comment (dorschw) [inline CHANGELOG.md:51]: was this one misplaced?

**Review by dorschw — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (israelpoli) [reply CHANGELOG.md:51]: No
It is only merged in the last week to the master.

File: CHANGELOG.md (hunk 1 above)
Comment (israelpoli) [reply CHANGELOG.md:32]: No
It is only merged in the last week to the master.

**Pull request merged**